project-management/project-management.md

```diff
@@ -44,14 +44,17 @@ Any Digital Preservation Staff may perform this process
 
 * Move the collection card to the list that represents the current status of the checklist items.  
     * When working on the Inventory checklist items move the card to the Inventory list.  
+    * Note, checking Export FTK reports item moves the card to Post-processing list automatically.  
+    * Note, checking Transfer FA components... item moves the card to Storage list automatically.  
+    *  Note, checking Add 2 year retention date... item moves the card to Storage list automatically.  
 * Add yourself to a collection card for a collection you are working on.  
 * Add a checklist item for each type and number of media in the Transfer checklist when more than one staff member is working on transferring a collection.  
     - [ ] 10 optical discs  
     - [ ] 1 hard drive  
 * Contact archivist to advise on transfer as needed.
 * Add a comment to the AU Trello card recommending FTK or workstation processing once all media has been transferred and digital preservation staff have reviewed the material.  
 * Add comments to describe deaccessions or transfer issues.  
-* Add a two year due date to the card once archivist description is complete.  
+* Add a 2 year due date to the Retain FTK Case custom field once archivist description is complete.  
```

Comment:
Unclear, maybe

"Add a due date to the Retain FTK Case custom field for 2 years after the archivist's description is complete"